S3/English/ASS_26.ass

```diff
@@ -0,0 +1,108 @@
+Style: Title,Arial,75,&H006CC2E9,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,100,0,0,1,2,2,2,10,10,280,1
+
+[Events]
+Format: Layer, Start, End, Style, Name, MarginL, MarginR, MarginV, Effect, Text
+Dialogue: 0,0:01:02.51,0:01:05.56,Title,,0,0,0,, {\pos(970,920)} {\b1\an5\pos(970,880)\fs120\frz4.5\t(0,100,1.5\fs45)}Episode 26:  {\c&H7990E7&}Frightening Halloween{\b0}
+Dialogue: 0,0:01:14.22,0:01:16.49,Default,,0,0,0,,The lord is already inside
+Dialogue: 0,0:01:16.87,0:01:18.60,Default,,0,0,0,,He has been waiting for a long time
+Dialogue: 0,0:01:19.09,0:01:19.81,Default,,0,0,0,,What
+Dialogue: 0,0:01:20.82,0:01:21.80,Default,,0,0,0,,Don't do that!
+Dialogue: 0,0:01:22.29,0:01:22.85,Default,,0,0,0,,Shut up
+Dialogue: 0,0:01:23.05,0:01:24.37,Default,,0,0,0,,I can't even hear the lines
+Dialogue: 0,0:01:27.23,0:01:27.69,Default,,0,0,0,,Who's that
+Dialogue: 0,0:01:28.24,0:01:28.91,Default,,0,0,0,,It's midnight
+Dialogue: 0,0:01:29.73,0:01:30.43,Default,,0,0,0,,I'll open the door
+Dialogue: 0,0:01:44.19,0:01:45.21,Default,,0,0,0,,I'm back
+Dialogue: 0,0:01:48.09,0:01:49.27,Default,,0,0,0,,Did I went to the wrong door
+Dialogue: 0,0:01:51.08,0:01:51.70,Default,,0,0,0,,Luis
+Dialogue: 0,0:01:52.37,0:01:52.95,Default,,0,0,0,,Ira
+Dialogue: 0,0:01:57.12,0:01:58.45,Default,,0,0,0,,My name is Luis
+Dialogue: 0,0:01:58.74,0:01:59.75,Default,,0,0,0,,Just call me  Lulu
+Dialogue: 0,0:02:00.24,0:02:01.07,Default,,0,0,0,,I always had a dream
+Dialogue: 0,0:02:01.18,0:02:02.32,Default,,0,0,0,,To become a film star
+Dialogue: 0,0:02:03.24,0:02:04.32,Default,,0,0,0,,But at my hometown
+Dialogue: 0,0:02:04.54,0:02:06.32,Default,,0,0,0,,The job is so boring
+Dialogue: 0,0:02:06.88,0:02:08.55,Default,,0,0,0,,So I decided to leave my home
+Dialogue: 0,0:02:08.92,0:02:10.07,Default,,0,0,0,,To find a job here
+Dialogue: 0,0:02:10.49,0:02:11.77,Default,,0,0,0,,80yuan a day, need a crowd actor
+Dialogue: 0,0:02:12.12,0:02:12.67,Default,,0,0,0,,Choose me
+Dialogue: 0,0:02:12.67,0:02:14.40,Default,,0,0,0,,I had experience of 12 films
+Dialogue: 0,0:02:15.03,0:02:16.40,Default,,0,0,0,,I'm the universal crowd actor
+Dialogue: 0,0:02:19.14,0:02:19.63,Default,,0,0,0,,My head!
+Dialogue: 0,0:02:23.84,0:02:24.75,Default,,0,0,0,,It' fine
+Dialogue: 0,0:02:25.12,0:02:27.06,Default,,0,0,0,,I couldn't be an actor if it was broken
+Dialogue: 0,0:02:27.28,0:02:27.67,Default,,0,0,0,,Hey
+Dialogue: 0,0:02:28.00,0:02:28.66,Default,,0,0,0,,Bro
+Dialogue: 0,0:02:29.12,0:02:29.76,Default,,0,0,0,,Beside your head
+Dialogue: 0,0:02:30.00,0:02:31.57,Default,,0,0,0,,Could you also take off other parts of your body?
+Dialogue: 0,0:02:32.22,0:02:34.30,Default,,0,0,0,,I... got caught by gangsters
+Dialogue: 0,0:02:36.12,0:02:36.80,Default,,0,0,0,,Ye...Yes
+Dialogue: 0,0:02:37.22,0:02:39.16,Default,,0,0,0,,I haven't become famous yet...
+Dialogue: 0,0:02:40.12,0:02:41.48,Default,,0,0,0,,I'm not willing to
```

Comment:
Need a better tl

Reply:
no idea at all, in the episode context I would've gone for "so unlucky" (since he just lost his head and didn't get the job) or "I'm not ready to die" (but he's already dead so maybe not this one), or just "I don't want to" as in work with the gangsters and risk his life

Reply:
> no idea at all, in the episode context I would've gone for "so unlucky" (since he just lost his head and didn't get the job) or "I'm not ready to die" (but he's already dead so maybe not this one), or just "I don't want to" as in work with the gangsters and risk his life

Yeah go for the meaning is better